main/app_main.c

```diff
@@ -53,12 +54,17 @@ const static char http_jpg_hdr[] =
 const static char http_pgm_hdr[] =
         "Content-type: image/x-portable-graymap\r\n\r\n";
 const static char http_stream_boundary[] = "--123456789000000000000987654321\r\n";
+const static char http_bitmap_hdr[] =
+        "Content-type: image/bitmap\r\n\r\n";
 
 static EventGroupHandle_t wifi_event_group;
 const int CONNECTED_BIT = BIT0;
 static ip4_addr_t s_ip_addr;
 static camera_pixelformat_t s_pixel_format;
 
+#define CAMERA_PIXEL_FORMAT CAMERA_PF_RGB565
+#define CAMERA_FRAME_SIZE CAMERA_FS_QQVGA
```

Comment:
Recently I've been thinking that rebuilding the whole project and the whole IDF each time I want to change camera configuration is not nice. OTOH, it's nice to have stuff configurable from a GUI. Maybe it's time we have a decent web server and configure everything through it? I've been looking at [ureq](https://github.com/solusipse/ureq), which is really small and easy to integrate, unfortunately it doesn't really have a way of sending (part of) the data in `NETCONN_NOCOPY` mode, which is essential when streaming bit chunks of framebuffer.
[civetweb](https://github.com/civetweb/civetweb) (MIT-licensed for of Mongoose) is another nice solution, although might take more time to integrate.
I guess i need to open a separate issue to discuss which embedded web server to use...